src/ui/component/ChainSelector/Modal.tsx

```diff
@@ -40,10 +38,23 @@ const ChainSelectorModal = ({
   const handleChange = (val: CHAINS_ENUM) => {
     onChange(val);
   };
-
+  const goToChainManagement = () => {
+    history.push({
+      pathname: '/settings/chain',
+      state: {
+        connection,
+      },
+    });
+  };
   const init = async () => {
-    setEnableChains(await wallet.getEnableChains());
-    setCurrentAccount(await wallet.syncGetCurrentAccount());
+    const savedChains = await wallet.getSavedChains();
+    const getSupportChains = await wallet.getSupportChains();
```

Comment:
Why we need supportChains?
This should be all chains right?

```typescript
import { CHAINS } from 'consts';
Object.values(CHAINS)
```

Reply:
So we don't  need to retrieve  chain from open api, but from local ?

Reply:
done